src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,105 @@
-import React from 'react';
 import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
```

Comment:
It's a good practice to order and group the imports. You can group external imports first, followed by internal imports. Also, ordering them alphabetically can make it easier to manage.